thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -53,8 +53,8 @@ struct TSetStorageGroupReq {
   1: required TStorageGroupSchema storageGroup
 }
 
-struct TDeleteStorageGroupReq {
-  1: required string storageGroup
+struct TDeleteStorageGroupsReq {
+  1: required list<string> prefixPathList
```

Comment:
Because there is only _deleteStorageGroups_  in the rpc interface at previous version, so I just restore it as it is in previous. Should I implement the interface of delete a single storage group besides as we discussed ?

![image](https://user-images.githubusercontent.com/82880298/167241359-d2e569ff-5237-4ad5-83cb-15aaa6dcffc7.png)

Reply:
Now a deleteStorageGroup is added.